src/System.Data.SqlClient/src/System/Data/Sql/SqlMetaData.cs

```diff
@@ -961,7 +1052,7 @@ public SqlBinary Adjust(SqlBinary value)
             {
                 byte[] rgbValue = value.Value;
                 byte[] rgbNewValue = new byte[MaxLength];
-                Buffer.BlockCopy(rgbValue, 0, rgbNewValue, 0, (int)MaxLength);
+                Array.Copy(rgbValue, rgbNewValue, (int)MaxLength);
```

Comment:
Why did you change this to Array.Copy here and the rest of the occurrences in this file? 
Isn't `Buffer.BlockCopy` more performant for byte array copy?

Reply:
The original framework implementation used Array.Copy. Should I switch it back to Buffer.BlockCopy?

Reply:
Yes. Please switch it back.